docs/arcaflow_scenarios.md

```diff
@@ -63,13 +63,45 @@ This scenario is based on the arcaflow [sysbench](https://github.com/akopytov/sy
 The purpose of this scenario is to create cpu pressure on a particular node of the Kubernetes/OpenShift cluster for a time span.
 To enable this plugin add the pointer to the scenario input file `scenarios/arcaflow/sysbench-cpu-hog/input.yaml` as described in the 
 Usage section.
-This scenarios takes the following input parameters:
+This scenario takes the following input parameters:
 
-- **kubeconfig :** string representation of the kubeconfig needed by the deployer to deploy the sysbench plugin in the target cluster
+- **kubeconfig :** *string* the kubeconfig needed by the deployer to deploy the sysbench plugin in the target cluster
 **Note:** this parameter will be automatically filled by kraken if the `kubeconfig_path` property is correctly set
-- **node_selector :**  key-value representation of the node label that will be used as `nodeSelector` by the pod to target a specific cluster node
-- **sysbench_cpumaxprime :** integer that defines the highest prime number during the test. Higher this value is, higher will be the time to find all the prime numbers
-- **sysbench_events :** integer that limits the maximum number of events that will be performed by sysbench, 0 removes the limit
-- **sysbench_runtime :** number of seconds the test will be run
-- **sysbench_forced_shutdown_time :** the number of seconds to wait before shutting down the benchmark after the defined run time
-- **sysbench_threads :** the number of threads on which the test will run
\ No newline at end of file
+- **node_selector :**  *key-value map* the node label that will be used as `nodeSelector` by the pod to target a specific cluster node
+- **sysbench_cpumaxprime :** *int* the highest prime number during the test. Higher this value is, higher will be the time to find all the prime numbers
+- **sysbench_events :** *int* the maximum number of events that will be performed by sysbench, 0 removes the limit
+- **sysbench_runtime :** *int* number of seconds the test will be run
+- **sysbench_forced_shutdown_time :** *int* the number of seconds to wait before shutting down the benchmark after the defined run time
+- **sysbench_threads :** *int* the number of threads on which the test will run
+
+##### kill-pod
```

Comment:
We might want to move the docs under https://github.com/redhat-chaos/krkn/blob/main/docs/pod_scenarios.md.